site/en/blog/third-party-scripts/index.md

```diff
@@ -0,0 +1,264 @@
+-   Not authored or influenced by an individual site owner.
+-   Widely used by a variety of sites.
+
+From helping to generate revenue (via ads) to providing better marketing opportunities (social media embeds), third-parties serve a wide variety of business objectives. Common categories of third-parties include the following:
+
+Source: [Third-parties by category](https://github.com/patrickhulce/third-party-web/#third-parties-by-category).
+
+<table>
+  <thead>
+    <tr>
+      <th><br>
+<strong>Category</strong></th>
+      <th><br>
+<strong>Definition</strong></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><br>
+Advertising</td>
+      <td><br>
+Scripts used for serving ads or measuring ad performance.</td>
+    </tr>
+    <tr>
+      <td><br>
+Video</td>
+      <td><br>
+Scripts that enable video player and streaming functionality.</td>
+    </tr>
+    <tr>
+      <td><br>
+Hosted libraries</td>
+      <td><br>
+A mixture of publicly hosted open source libraries </td>
+    </tr>
+    <tr>
+      <td><br>
+Content</td>
+      <td><br>
+Scripts from content providers or publishing-specific affiliate tracking.</td>
+    </tr>
+    <tr>
+      <td><br>
+Customer Success</td>
+      <td><br>
+Scripts from customer support/marketing providers that offer chat and contact solutions.</td>
+    </tr>
+    <tr>
+      <td><br>
+Hosting</td>
+      <td><br>
+Scripts from web hosting platforms</td>
+    </tr>
+    <tr>
+      <td><br>
+Marketing</td>
+      <td><br>
+Scripts from marketing tools that add popups, newsletters, etc</td>
+    </tr>
+    <tr>
+      <td><br>
+Social</td>
+      <td><br>
+Scripts that enable social features</td>
+    </tr>
+    <tr>
+      <td><br>
+Tag Manager</td>
+      <td><br>
+Scripts that load many other scripts and initiate many tasks.</td>
+    </tr>
+    <tr>
+      <td><br>
+Analytics</td>
+      <td><br>
+Scripts that measure or track users and their actions. </td>
+    </tr>
+    <tr>
+      <td><br>
+Cookie Consent Platform</td>
+      <td><br>
+Scripts that allow sites to obtain the user consent (GDPR, ePR, CCPA) in an informed and transparent way.</td>
+    </tr>
+    <tr>
+      <td><br>
+Utility</td>
+      <td><br>
+Scripts that are developer utilities (API clients, site monitoring, fraud detection, etc.</td>
+    </tr>
+    <tr>
+      <td><br>
+Other</td>
+      <td><br>
+Miscellaneous scripts delivered via a shared origin with no precise category or attribution.</td>
+    </tr>
+  </tbody>
+</table>
+
+These third-party scripts and libraries allow web developers to leverage tried and tested solutions to implement standard features instead of re-inventing the wheel. Thus, third-parties reduce development time and help businesses launch or upgrade their products faster. It is no wonder then that more than [94% of all websites](https://almanac.httparchive.org/en/2021/third-parties#fig-5) on desktop and mobile use third-parties.
```

Comment:
```suggestion
These third-party scripts and libraries allow web developers to use tried and tested solutions to implement standard features instead of reinventing the wheel. Thus, third-parties reduce development time and help businesses launch or upgrade their products faster. It is no wonder then that more than [94% of all websites](https://almanac.httparchive.org/en/2021/third-parties#fig-5) on desktop and mobile use third-parties.
```

Reply:
2022 almanac is coming out in October, so maybe we can revise this figure in the future. Just a thought.

Reply:
👍🏻